### DISCUSSION TITLE
How do you handle complex DFDs?

### DISCUSSION THREAD

**Discussion by srcnuzn** [Threagile/threagile#95] [General]
In other tools like OWASP Threat Dragon, I can create multiple data-flow-diagrams (DFDs) focusing on different aspects of the architecture. This makes it easier to understand the DFDs. In Threagile, though, I only see the option to create one DFD per .yaml file. How do you work with that? Do you create one large DFD or do you have multiple .yaml files?

**Comment by ezavgorodniy** [comment]
Is DFD stand for data flow diagram? 
When I started using Threagile - I had the same question and even added somewhere in TODO list to allow somehow filter of what I'd like to show on the diagram to be able to generate different data flow diagram from the one model.

The way how my current company is doing right now - small models per each project (with some duplication of course)

**Reply by srcnuzn** [reply]
Yes, DFD = data-flow-diagram. I've edited my post
